app/controllers/applications_controller.rb

```diff
@@ -42,6 +46,17 @@ def select
     redirect_to event_applications_path(@event), notice: "Cool! Changes saved."
   end
 
+  def confirm
+    if Time.now > sent_on.5.days.from_now
```

Comment:
There are two things here that wouldn't work:

1. What's `sent_on`? If it's supposed to be a column, we forgot to add the migration
2. Even if `sent_on` existed (and I guess it would be a `Date` or `Time`), I'm relatively sure it wouldn't have the `5` method (because that method name is illegal, it would be impossible to define 😂). We should discuss what are we trying to achieve here and find out how to do it 😸 

Side note: even if we don't have a specific test for this feature (maybe we should add it), our old tests told us about `2.`! https://travis-ci.org/rubymonsters/rgbapp/builds/331981354#L752